inductiva/mixins/file_manager.py

```diff
@@ -0,0 +1,230 @@
+"""Mixin for file management.
+
+Methods to manage files and directories, within your local computer in
+order to prepare them for the simulation process. This mixin can be
+used to copy files and directories, and render template files with
+the given set of arguments.
+"""
+from typing import Union
+import pathlib
+import shutil
+import glob
+import io
+import os
+import re
+
+from absl import logging
+import jinja2
+
+import inductiva
+
+TEMPLATE_EXTENSION = ".jinja"
+SUFFIX_SEPARATOR = "__"
+
+
+class FileManager:
+    """Class for file management."""
+    DEFAULT_ROOT_DIR = "input_dir"
+    __root_dir = None  #pylint: disable=invalid-name
+
+    def __check_root_dir(self):
+        """Set default root folder if unset."""
+        if self.__root_dir is None:
+            self.set_root_dir(self.DEFAULT_ROOT_DIR)
+
+    def set_root_dir(self, root_dir=None):
+        """Set a root directory for the file manager.
+        
+        All files managed through the manager will be
+        placed inside a newly created folder with the given
+        name. If a folder with the same name already exists,
+        the newly created one will be appended with the
+        "#1" suffix. If folders having the "#N" suffix already
+        exist, the given name will be appended with "#{N+1}"
+        where N is determined by the largest existing N.
+
+        Args:
+            root_dir: Path to the root directory.
+                If None, an error is raised.
+                If the directory already exists. A new directory
+                name is created following a sequence of #1, #2, #3, etc."""
+
+        if root_dir is None:
+            raise ValueError("Given root directory cannot be None")
+        elif os.path.isdir(root_dir):
+            if os.getenv("INDUCTIVA_DISABLE_FILEMANAGER_AUTOSUFFIX", False):  #pylint: disable=invalid-envvar-default
```

Comment:
There's a bug here (my bad).
env vars are always str. we need to convert correctly to a boolean.

in `inductiva.utils`, add a helper method to get boolean env vars:

```python
from distutils.util import strtobool

def getenv_bool(varname, default):
    return bool(strtobool(os.getenv(varname, default)))
```


```suggestion
# no need to pylint disable
            if utils.getenv_bool("INDUCTIVA_DISABLE_FILEMANAGER_AUTOSUFFIX", False):
```